dspace-api/src/main/java/org/dspace/content/CommunityServiceImpl.java

```diff
@@ -379,17 +390,29 @@ public void addCollection(Context context, Community community, Collection colle
 
     @Override
     public Community createSubcommunity(Context context, Community parentCommunity)
-        throws SQLException, AuthorizeException {
+            throws SQLException, AuthorizeException {
         return createSubcommunity(context, parentCommunity, null);
     }
 
+
     @Override
     public Community createSubcommunity(Context context, Community parentCommunity, String handle)
-        throws SQLException, AuthorizeException {
+            throws SQLException, AuthorizeException {
+        return createSubcommunity(context, parentCommunity, handle, null);
+    }
+
+    @Override
+    public Community createSubcommunity(Context context, Community parentCommunity, String handle,
+                                        UUID uuid) throws SQLException, AuthorizeException {
         // Check authorisation
         authorizeService.authorizeAction(context, parentCommunity, Constants.ADD);
 
-        Community c = create(parentCommunity, context, handle);
+        Community c;
+        if (uuid != null) {
+            c = create(parentCommunity, context, handle, uuid);
+        } else {
+            c = create(parentCommunity, context, handle);
+        }
```

Comment:
minon thing, this can be eventually simplified as the 4-params method always work